kubernetes/loculus/templates/ingressroute.yaml

```diff
@@ -45,6 +45,18 @@ spec:
     replacement: "https://$1"
     permanent: true
 ---
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: cors-forward
```

Comment:
forward doesn't seem the best name if as I understand it's not forwarding the CORS bits (just adding them afresh)

Reply:
Ah, but maybe this does something different sorry!

Reply:
(we do it differently in https://github.com/loculus-project/loculus/blob/6ff96d1eb5ada563dcc46ad3d96189fe60716956/kubernetes/loculus/templates/lapis-ingress.yaml#L6 )